packages/server/graphql/public/types/CreateTaskPayload.ts

```diff
@@ -4,13 +4,20 @@ import errorFilter from '../../errorFilter'
 import {CreateTaskPayloadResolvers} from '../resolverTypes'
 
 export type CreateTaskPayloadSource = {
-  taskId: string
+  error?: Error
+  taskId?: string
   notificationIds?: string[]
 }
 
 const CreateTaskPayload: CreateTaskPayloadResolvers = {
+  error: async ({error}) => {
+    if (!error) return null
+    return {
+      message: error.message
```

Comment:
Although the error already has the correct shape, without this resolver the error member in source would not be used as source here, but treated as if it was thrown by graphql.